### PR TITLE
Add Travis-CI to repo + Update tox to run tests

### PR DESCRIPTION
**Pull request by linuxlewis**
- Update settings.py to handle different test settings for versions of
  django
- Update tox.ini to run the tests for different environments
- Add .travis.yml to configure ci

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+install: "pip install tox"
+# command to run tests
+script: tox -r
+env:
+  - TOXENV=flake8
+  - TOXENV=py27
+  - TOXENV=py33
+  - TOXENV=py34
+addons:
+  postgresql: "9.3"
```
Comment (troygrosfield) [inline .travis.yml:12]: Version bump to latest version of postgres:

```
postgresql: "9.4"
```

**Comment by troygrosfield** [comment]
Any updates on this?

**Comment by mpasternak** [comment]
Ping! I'm interested, too. Just updated my branch, ran the tests, 10 failures and I'm like "WTF?!" (= where's the fault?!)

**Comment by linuxlewis** [comment]
@mpasternak I have some changes locally I haven't pushed up yet. The issues I was running into were related to tox and django test runner. I reorganized the tests and added pytest and things are looking promising.

**Comment by troygrosfield** [comment]
Looking forward to getting these changes integrated as well as the broken version that it's currently in pypi updated (management commands don't exist in pypi version).

**Pull request closed without merging**